src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Card-Strings.json

```diff
@@ -741,7 +741,7 @@
   },
   "stsjorbsmod:Siphon": {
     "NAME": "Siphon",
-    "DESCRIPTION": "Deal !D! damage. NL If this kills an enemy, heal !M! HP."
+    "DESCRIPTION": "Deal !D! damage. NL If this kills an enemy, heal !M! HP and gain !stsjorbsmod:MetaMagic! Max HP."
```

Comment:
I'm trying to imagine if there's any functional difference between heal then gain max hp and gain max hp then heal. I don't see any.

Reply:
I considered that as well, I think they're probably the same as well.